src/referentielijsten/api/forms.py

```diff
@@ -0,0 +1,21 @@
+from django import forms
+
+from import_export.forms import ConfirmImportForm, ImportForm
+
+from .models import Tabel
+
+
+class ItemImportForm(ImportForm):
+    selected_tabel = forms.ModelChoiceField(
+        queryset=Tabel.objects.all(),
+        label="Tabel",
+        required=False,
+        help_text="Selecteer een tabel om toe te passen op alle geïmporteerde items, "
```

Comment:
this should mention that the table_id in the csv will be overwritten if selected in the form.